dashboard/app/models/course_offering.rb

```diff
@@ -36,8 +36,8 @@ class CourseOffering < ApplicationRecord
   belongs_to :self_paced_pl_course_offering, class_name: 'CourseOffering', optional: true
 
   validates :category, acceptance: {accept: Curriculum::SharedCourseConstants::COURSE_OFFERING_CATEGORIES, message: "must be one of the course offering categories. Expected one of: #{Curriculum::SharedCourseConstants::COURSE_OFFERING_CATEGORIES}. Got: \"%{value}\"."}
-  validates :curriculum_type, acceptance: {accept: Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values, message: "must be one of the course offering curriculum types. Expected one of: #{Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values}. Got: \"%{value}\"."}
-  validates :marketing_initiative, acceptance: {accept: Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.to_h.values, message: "must be one of the course offering marketing initiatives. Expected one of: #{Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.to_h.values}. Got: \"%{value}\"."}
+  validates :curriculum_type, acceptance: {accept: Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values, message: "must be one of the course offering curriculum types. Expected one of: #{Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values}. Got: \"%{value}\"."}, allow_blank: true
```

Comment:
Was there a reason you went with `allow_blank` instead of `allow_nil`? I believe [allow_nil](https://edgeguides.rubyonrails.org/active_record_validations.html#allow-nil) would be slightly cleaner so that we're sure we avoid a situation where we have both empty strings and `nil` values in this column

Reply:
Ill make this change, I was not sure if we would like to account for both since there are some empty strings in the db now.

Reply:
Ah, thanks for looking! Do you know what fields those empty strings are for? I think we want to avoid that for these particular fields, but I suppose that sometimes there's a "real" distinction we'd want to make between `nil` and `''` and allow for both

Reply:
No need to spend more time investigating my question! Just curious. Once the allow_nil change is made, feel free to merge without a re-approval

Reply:
Actually im sorry im taking a look now and I dont see any fields set to empty string. I confused myself during testing this and saw empty strings in the db before making the change to nil. Sounds good thanks!

Reply:
No worries, makes sense! Thanks for circling back